### PR TITLE
February pull 3

### PR DESCRIPTION
**Pull request by DandelionSprout**
I needed a good excuse to background-binge *My Knight and Me*, so I added some more lists and list stuff.

* Added ~28 lists.
* Added 1 new syntax.
* Took care of #1373.

### CODE REVIEW

**Comment by collinbarrett** [comment]
[Looks like](https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=2975&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=27) maybe a typo somewhere.

File: data/FilterList.json [hunk 1]
```diff
@@ -21146,5 +21146,267 @@
...
+    "id": 2069,
+    "description": "Peter Lowe's (as of August 2018) colleague Vladimir Pissarev (who is part of the Security team in his company) mentioned to him that he runs a public DNS server that not only blocks ads and trackers, but also malware and viruses as well.",
+    "homeUrl": "https://itsecurity.ee/",
+    "name": "Vladimir Pissarev's DNS Server",
+    "syntaxId": 22,
+    "viewUrl": "91.239.26.140"
+  },
+  {
+    "id": 2070,
+    "description": "I wanted an easy solution to blocking ads and tracking on all my devices and AdGuard DNS didn’t block quite enough for me, so I decided to run my own AdGuard Home server. Sharing is caring, which is why I’ve decided to make my server available for everyone! Links that didn't fit in the Subscribe button URL texts, include 2a03:b0c0:3:d0::5f18:6001"
```
Comment (collinbarrett) [inline data/FilterList.json]: I think missing a comma at the end of this line.

Comment (DandelionSprout) [reply data/FilterList.json]: The syntax checker in "Files changed" seemed to point out the same problem as well, so I've tried to fix it now.

**Comment by DandelionSprout** [comment]
I can't find any more spelling errors in my pull now, so I can only presume the build errors must be due to whatever else...

File: data/FilterListTag.json [hunk 2]
```diff
@@ -10086,5 +10086,185 @@
...
+    "filterListId": 2058,
+    "tagId": 24
+  },
+  {
+    "filterListId": 2059,
+    "tagId": 24
+  },
+    {
+    "filterListId": 205,
+    "tagId":
```
Comment (collinbarrett) [inline data/FilterListTag.json]: The build error [does give a line number](https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=2976&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=27) if you know what you are looking for. I'll try to figure out a way to make that easier to find.

But, regardless, I think this lack of tagId is an issue now.

Comment (DandelionSprout) [reply data/FilterListTag.json]: I noticed the line number, but didn't know which file to look for it in.

I'll try to fix it now.

Comment (collinbarrett) [reply data/FilterListTag.json]: Yeah, the error messages could definitely be improved...

Comment (DandelionSprout) [reply data/FilterListTag.json]: Okay, I think I saw the real error now. This sure became a bit strange. Give me another minute.

**Pull request merged**